core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -23,9 +23,12 @@ import { State, StateBackendDict, StateObjectFactory }
   from 'domain/state/StateObjectFactory';
 import INTERACTION_SPECS from 'interactions/interaction_specs.json';
 import constants from 'assets/constants';
+import { Misconception } from 'domain/skill/MisconceptionObjectFactory';
+
+type InteractionId = keyof typeof INTERACTION_SPECS;
```

Comment:
I just realized that this is already defined in https://github.com/oppia/oppia/blob/develop/core/templates/pages/interaction-specs.constants.ts#L25

Reply:
Done